clidocstool_man_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 cli-docs-tool authors
```

Comment:
Was this new code, or was this the code migrated from docker/cli? If this was the code that was migrated, perhaps it should be 2017 🤔 (otherwise 2021 - being the year that you opened the PR is ok)